r/rsparkling/tests/test_h2o_name.R

```diff
@@ -0,0 +1,25 @@
+# Test of transformations from dataframe to h2o frame and from h2o frame back to dataframe
+
+library(testthat)
+library(rsparkling)
+library(dplyr)
+
+# Testing passing name to a dataframe
+test_h2o_name = function(){
+
+   mtcars_tbl <- copy_to(sc, mtcars, overwrite = TRUE)
+   mtcars_hf_name <- as_h2o_frame(sc, mtcars_tbl, name = "frame1")
+
+   expect_equal(h2o.getId(mtcars_hf), "frame1")
+
+
+}
+
+#If Spark is not installed on the machine, otherwise it will not
+spark_install(version = "1.6.2")
```

Comment:
Note: The test should install nothing. We need to make sure that the env is ready (we can pass `SPARK_HOME` to this process)